flipperdevices/flipperzero-firmware
# F6: USB glue cleanup and fixes

# What's new

- F6: proper USB-CDC integration with furi-hal-vcp, fix lost data on buffer overflow
- F6: USB code cleanup and move to usb-glue folder
- F6: USB Descriptor adjustments: power, CDC and endpoint sizes
- F6: USB smaller buffers
- F6: bootloader keeps USB lines pulled to GND till startup complete
- Cli: fix missing motd with minicom

# Verification 

- Compile and upload bootloader and firmware
- Send a large block of data (> 600b) to cdc, you should see an echo of all data
- Check on reboot USB behavior. Bus should be properly reseted
- USB disconnect on screen streaming doesn't cause lockups anymore
- Minicom now doesn't skip motd

# Checklist (do not modify)
